### PR TITLE
Build release build on tags.

### LINKED ISSUE #10067: b_ems does not create release binary on circleci for tags

### PR DESCRIPTION
**Pull request by chriseth**
Fixes https://github.com/ethereum/solidity/issues/10067

### CODE REVIEW

File: scripts/travis-emscripten/build_emscripten.sh [hunk 1]
```diff
@@ -42,8 +42,18 @@ fi
 
 WORKSPACE=/root/project
 
-echo -en 'travis_fold:start:compiling_solidity\\r'
 cd $WORKSPACE
+
+# shellcheck disable=SC2166
+if [[ "$CIRCLE_BRANCH" == release -o -n "$CIRCLE_TAG" -o -n "$FORCE_RELEASE" -o "$(git tag --points-at HEAD 2>/dev/null)" == v* ]]
```
Comment (leonardoalt) [inline scripts/travis-emscripten/build_emscripten.sh]: Will this work with these `CIRCLE` vars?

Comment (chriseth) [reply scripts/travis-emscripten/build_emscripten.sh]: Yes, on circleci. See also https://github.com/ethereum/solidity/pull/10072

File: scripts/travis-emscripten/build_emscripten.sh [hunk 2]
```diff
@@ -42,8 +42,18 @@ fi
 
 WORKSPACE=/root/project
 
-echo -en 'travis_fold:start:compiling_solidity\\r'
 cd $WORKSPACE
+
+# shellcheck disable=SC2166
+if [[ "$CIRCLE_BRANCH" = release || -n "$CIRCLE_TAG" || -n "$FORCE_RELEASE" || "$(git tag --points-at HEAD 2>/dev/null)" == v* ]]
```
Comment (leonardoalt) [inline scripts/travis-emscripten/build_emscripten.sh:48]: The other scripts don't seem to have the `git` part. Why add here and not there?

Comment (chriseth) [reply scripts/travis-emscripten/build_emscripten.sh:48]: I took that from a third script, and I think it is a good idea so that it also works locally.

**Review by leonardoalt — APPROVED** [review]

**Pull request merged**